pkg/controller/history/controller_history.go

```diff
@@ -252,6 +252,13 @@ func (rh *realHistory) CreateControllerRevision(parent metav1.Object, revision *
 		clone.Name = ControllerRevisionName(parent.GetName(), hash)
 		created, err := rh.client.AppsV1().ControllerRevisions(parent.GetNamespace()).Create(clone)
 		if errors.IsAlreadyExists(err) {
+			existed, err := rh.client.AppsV1().ControllerRevisions(parent.GetNamespace()).Get(clone.Name, metav1.GetOptions{})
+			if err != nil {
+				return nil, err
+			}
+			if bytes.Equal(existed.Data.Raw, clone.Data.Raw) {
+				return existed, err
```

Comment:
`existed, err` -> `existed, nil` to make an explicit statement here.
or `exists, nil` as per the above comment about tense.